inference-engine/tests/functional/plugin/cpu/single_layer_tests/mat_mul.cpp

```diff
@@ -18,80 +18,88 @@ enum class MatMulNodeType {
     FullyConnected
 };
 
-using MatMulLayerTestParams = std::tuple<std::pair<SizeVector, SizeVector>,
-                                         Precision,
-                                         helpers::InputLayerType,
-                                         bool,
-                                         bool>;
-
-using MatMulLayerCPUTestParamSet = std::tuple<MatMulLayerTestParams,
+using MatMulLayerCPUTestParamSet = std::tuple<MatMulLayerTestParamsSet,
                                               MatMulNodeType,
                                               fusingSpecificParams>;
 
 class MatMulLayerCPUTest : public testing::WithParamInterface<MatMulLayerCPUTestParamSet>,
                                 virtual public LayerTestsUtils::LayerTestsCommon, public CpuTestWithFusing {
 public:
-    static std::string getTestCaseName(testing::TestParamInfo<MatMulLayerCPUTestParamSet> obj) {
-        MatMulLayerTestParams basicParamsSet;
-        fusingSpecificParams fusingParams;
+    static std::string getTestCaseName(const testing::TestParamInfo<MatMulLayerCPUTestParamSet>& obj) {
+        MatMulLayerTestParamsSet basicParamsSet;
         MatMulNodeType nodeType;
-        std::tie(basicParamsSet, nodeType, fusingParams) = obj.param;
+        fusingSpecificParams fusingParams;
 
-        std::pair<SizeVector, SizeVector> IS;
-        SizeVector isA, isB;
-        bool transpA, transpB;
-        Precision prec;
-        helpers::InputLayerType typeB;
-        std::tie(IS, prec, typeB, transpA, transpB) = basicParamsSet;
-        isA = IS.first; isB = IS.second;
+        std::tie(basicParamsSet, nodeType, fusingParams) = obj.param;
 
         std::ostringstream result;
         result << (nodeType == MatMulNodeType::MatMul ? "MatMul_" : "FullyConnected_");
-        result << "IS_A=" << CommonTestUtils::vec2str(isA) << "_";
-        result << "IS_B=" << CommonTestUtils::vec2str(isB) << "_";
-        result << "Transp_A=" << transpA << "_";
-        result << "Transp_B=" << transpB << "_";
-        result << "Prec=" << prec << "_";
-        result << "typeB=" << typeB;
-
+        result << LayerTestsDefinitions::MatMulTest::getTestCaseName(
+            testing::TestParamInfo<LayerTestsDefinitions::MatMulLayerTestParamsSet>(basicParamsSet, 0));
         result << CpuTestWithFusing::getTestCaseName(fusingParams);
 
+        /* exclude "ENFORCE_BF16, NO" to avoid skipping the test because of "BF16"  */
```

Comment:
Maybe we can just push empty additionalConfig (instead of "ENFORCE_BF16, NO") in case we don't wont to force BF16?

Reply:
Done